main.go

```diff
@@ -74,12 +76,17 @@ func getEnvAsBool(name string, defaultValue bool) bool {
 func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
+	var monitoringNS string
 	var probeAddr string
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+
+	flag.StringVar(&monitoringNS, "monitoring-namespace", "",
```

Comment:
I'm wondering how we'd allow users to monitor their components in odh-core installs... If the ODH operator is what is installing the odh model controller, how are users going to override it?

Reply:
Perhaps we default to creating them in the same namespace?

Reply:
We settled on using kfdef parameters for configuration stuff.